redditapi.js

```diff
@@ -1,9 +1,9 @@
 export default {
   search: function(searchTerm, searchLimit, sortBy) {
-    fetch(`http://www.reddit.com/search.json?q=
+    return fetch(`http://www.reddit.com/search.json?q=
```

Comment:
This could be an anonymous function, if you fancy, but this works great anyway, just a matter of preference